examples/data-objects/monaco/src/view.tsx

```diff
@@ -68,7 +68,7 @@ export const MonacoView: React.FC<IMonacoViewProps> = (props: IMonacoViewProps)
         const codeModel = monaco.editor.createModel(sharedString.getText(), "typescript");
         const outputModel = monaco.editor.createModel("", "javascript");
         const codeEditor = monaco.editor.create(
-            viewElementRef.current,
+            viewElementRef.current!,
```

Comment:
Non-null assertions get a lint error, and they're generally good to avoid when possible since it's easy for these assertions to be untruthful (or at least we should have a comment explaining how we know the value is non-null with 100% certainty).

Another way to avoid the build error is to validate that the value is in fact non-null using an assertion/throw to type-narrow.  Here's an example of us using this strategy for newItemTextInputRef: https://github.com/microsoft/FluidFramework/blob/main/examples/data-objects/todo/src/Todo/TodoView.tsx#L52